website/docs/docs/cloud/secure/bigquery-psc.md

```diff
@@ -31,6 +31,9 @@ import PrivateLinkSLA from '/snippets/_private-connection-SLA.md';
 
 <PrivateLinkSLA />
 
+## (Optional) Generate Bigquery Credentials
+You may already have credentials set up for your datasets. If not, you can follow [this](docs/guides/bigquery-qs.md#generate-bigquery-credentials) guide to generate credentials. 
```

Comment:
I don't think I did the linking right since it links to the guide but not the specific step I was trying to point to (according to the [preview](https://docs-getdbt-com-git-nli-small-updates-to-bigquery-psc-dbt-labs.vercel.app/docs/cloud/secure/bigquery-psc#optional-generate-bigquery-credentials)). Is there any guidance on how to do that?